### PR TITLE
Fixed Gemfile

### PR DESCRIPTION
**Pull request by abhishekt98**
Ref: https://github.com/bigbinary/neeto-compliance/issues/572

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -29,3 +29,17 @@ group :development do
...
+
+    # neeto-commons-backend gem for methods and modules that are used across all neeto products.
+    gem "neeto-commons-backend"
+
+    # For neeto audit
+    gem "neeto-compliance"
+
+  end
+end
+
```
Comment (sandip-mane) [inline Gemfile:44]: @neerajdotname this is a public-facing gem, do we need these dependencies for it? When someone uses our gem, their Rails will install neeto gems too. (Does not seem correct)

I thought of mentioning this in the README, but it's public-facing repo hence not a good idea to mention it there.

**Comment by abhishekt98** [comment]
@sandip-mane 

> When someone uses our gem, their Rails will install neeto gems too

The gems added in the Gemfile of a gem won't be installed along with the gem, only the gems defined as a dependency in the gemspec file will be installed along with the gem. But here the repo seems public, in that case we are actually exposing our private gem server token to the public

cc: @unnitallman

**Comment by unnitallman** [comment]
@sandip-mane Abhishek is right. Do we really need the repo to be public. Just the gem needs to be publicly installable, right?

**Comment by sandip-mane** [comment]
Moving discussion to https://github.com/bigbinary/neeto-bugtrap-web/issues/217

**Pull request closed without merging**